### PR TITLE
#154 Add ability to setup levels connection

### PR DESCRIPTION
**Pull request by AndriyCh1**
The commented code is written for domain connections but it is not finished yet

### CODE REVIEW

**Review by nikita-remeslov — COMMENTED** [review]
Can you leave a comment in the components, that other devs understand that we need this commented code?

**Comment by AndriyCh1** [comment]
> 

ok, i`ll do it

File: frontend/src/components/admin/path-connection/levels-list/styles.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
+@import 'assets/scss/utils';
+
+.level-item__button {
+  border-color: $gu-blue !important;
+  cursor: pointer;
```
Comment (karyna-kh) [inline frontend/src/components/admin/path-connection/levels-list/styles.scss]: use bootstrap classes for this: 'cursor-pointer'

File: frontend/src/components/admin/path-connection/list-item/styles.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
+@import 'assets/scss/utils';
+
```
Comment (karyna-kh) [inline frontend/src/components/admin/path-connection/list-item/styles.scss]: this css file and [frontend/src/components/admin/path-connection/levels-list/styles.scss](url) are identical,
you should use only one css file

File: frontend/src/components/admin/path-connection/pair-group/styles.scss [hunk 3]
```diff
@@ -0,0 +1,11 @@
+.paired-group {
+  width: 100%;
```
Comment (karyna-kh) [inline frontend/src/components/admin/path-connection/pair-group/styles.scss]: use bootstrap classes for this: 'w-100'

File: frontend/src/components/admin/path-connection/pair-item/styles.scss [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+.pair-delete {
+  background-color: $white;
+  color: $gu-purple;
+  border: 0.1em solid $purple-light;
+  border-radius: 50%;
+  width: 1.5em;
+  height: 1.5em;
+  padding: 0.2em;
+  margin-left: 2em;
+  cursor: pointer;
```
Comment (karyna-kh) [inline frontend/src/components/admin/path-connection/pair-item/styles.scss]: use bootstrap classes for this: 'cursor-pointer'

File: frontend/src/components/admin/path-connection/pair-item/styles.scss [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+}
+
+.pair-action-wrapper {
+  position: absolute;
```
Comment (karyna-kh) [inline frontend/src/components/admin/path-connection/pair-item/styles.scss]: use bootstrap classes for this: 'position-absolute'

File: frontend/src/components/admin/path-connection/pair-item/styles.scss [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+  right: 0;
+  right: -2em;
+  top: 50%;
+  transform: translateY(-50%);
+  display: none;
```
Comment (karyna-kh) [inline frontend/src/components/admin/path-connection/pair-item/styles.scss:19]: use bootstrap classes for this: 'd-none'

File: frontend/src/components/admin/path-connection/pair-item/styles.scss [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+  z-index: 100;
+}
+
+.pair-item {
+  position: relative;
```
Comment (karyna-kh) [inline frontend/src/components/admin/path-connection/pair-item/styles.scss]: use bootstrap classes for this: 'position-relative'

File: frontend/src/components/admin/path-connection/pair-item/styles.scss [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+    }
+  }
+}
+
+.pair-delete:hover {
+  background-color: $purple-light;
+}
+
+.pair-item-element {
+  position: relative;
```
Comment (karyna-kh) [inline frontend/src/components/admin/path-connection/pair-item/styles.scss]: use bootstrap classes for this: 'position-relative'

**Review by nikita-remeslov — APPROVED** [review]

File: frontend/src/components/admin/path-connection/pair-item/styles.scss (hunk 6 above)
Comment (AndriyCh1) [reply frontend/src/components/admin/path-connection/pair-item/styles.scss:19]: it brokes the element

**Comment by nikita-remeslov** [comment]
@AndriyCh1 Conflicts

**Pull request merged**